js/youre_square.js

```diff
@@ -0,0 +1,10 @@
+// https://www.codewars.com/kata/youre-a-square/train/javascript
+
+var isSquare = function(n){
+  if(Math.sqrt(n) !== NaN && Number.isInteger(Math.sqrt(n))){
```

Comment:
What is the purpose of `Math.sqrt(n) !== NaN`? Will it ever be `false` :smile: ?

Aside from that, nice work :smile: 

And the same comment applies as in #12 - can this be written as a single `return`?

Reply:
I've made it shorter.